doc/examples/bulk.rst

```diff
@@ -141,11 +141,12 @@ and fourth operations succeed.
    'upserted': [],
    'writeConcernErrors': [],
    'writeErrors': [{'code': 11000,
-                    'errmsg': '...E11000...duplicate key error...',
+                    'errmsg': 'E11000 duplicate key error collection: '
+                              'bulk_example.test index: _id_ dup key: { _id: 1 }',
```

Comment:
I think the previous version with `...` is more robust to changes in the server's errmsg.

Reply:
Done